src/Microsoft.AspNetCore.Server.HttpSys/NativeInterop/ComNetOS.cs

```diff
@@ -15,11 +15,7 @@ static ComNetOS()
         {
             var win8Version = new Version(6, 2);
 
-#if NETSTANDARD1_3
-            IsWin8orLater = (new Version(RuntimeEnvironment.OperatingSystemVersion) >= win8Version);
-#else
             IsWin8orLater = (Environment.OSVersion.Version >= win8Version);
```

Comment:
How do you know you're running on Windows, though? What does this do in Linux 8.3?